airflow/cli/commands/version_command.py

```diff
@@ -16,9 +16,8 @@
 # under the License.
 """Version command"""
 import airflow
-from airflow import settings
 
 
 def version(args):
     """Displays Airflow version at the command line"""
-    print(settings.HEADER + "  v" + airflow.__version__)
+    print(f"{airflow.__version__}")
```

Comment:
```suggestion
    print(airflow.__version__)
```

Wait a second.. why are we now still using f string if it is only one variable without any extra string attached to it? 😁

Reply:
Good catch！

Reply:
Fixed 😅

Reply:
😅